server/requirements.txt

```diff
@@ -1 +1,3 @@
-spacetrack==0.16.0
+boto3
+spacetrack
```

Comment:
Was there a reason we require spacetrack==0.16.0?

Reply:
I don't think so. I suspect it was c/p from an earlier script I was working with.